src/main/java/com/kodilla/ecommercee/service/ProductService.java

```diff
@@ -12,7 +12,6 @@
 @Service
 @RequiredArgsConstructor
 public class ProductService {
-    @Autowired
```

Comment:
so import org.springframework.beans.factory.annotation.Autowired is not needed anymore...